modules/classifications/auxiliaries/define/reponse.go

```diff
@@ -29,9 +29,8 @@ func (auxiliaryResponse auxiliaryResponse) GetError() error {
 func newAuxiliaryResponse(classificationID types.ID, error error) helpers.AuxiliaryResponse {
 	if error != nil {
 		return auxiliaryResponse{
-			Success:          false,
-			Error:            error,
-			ClassificationID: classificationID,
```

Comment:
required for scenario where one tries to define a new auxiliary but one already exists so need to return the already existing classification id

Reply:
will revert this then